app/src/common/img/ic-warning.svg

```diff
@@ -0,0 +1,11 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="13" viewBox="0 0 16 13">
+    <g fill="none" fill-rule="evenodd">
+        <g fill="#F36C4A">
+            <g>
+                <g>
+                    <path d="M16 14L8 1 0 14h16zm-7-1H7v-2h2v2zm0-4H7V6h2v3z" transform="translate(-230 -247) translate(230 245) translate(0 1)"/>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
This svg image already exist as `error-inline`, please don't duplicate it. Check `ModalNote` component where same icon used.